recipes/tagine.md

```diff
@@ -0,0 +1,53 @@
+# Lubes' Vegetable Tagine
+
+![Difficulty Badge](https://img.shields.io/badge/Difficulty-60%25-orange.svg) 
+![Speed Badge](https://img.shields.io/badge/Speed-2hr-orange.svg)
+
+
+![Vegetarian Friendly Badge](https://img.shields.io/badge/Vegetarian-True-brightgreen.svg)
+![Vegan Friendly Badge](https://img.shields.io/badge/Vegan-True-brightgreen.svg)
```

Comment:
So, can we have a badge that allows for exploitation of bees, but not cattle? For the record, I used to be a shrimp-and-honey eating vegan on the grounds that--seriously--who really cares about anything with a brain that's smaller than a pinhead.